apps/fz_wall/lib/fz_wall/cli/live.ex

```diff
@@ -67,11 +67,17 @@ defmodule FzWall.CLI.Live do
   end
 
   @doc """
-  Adds device ip to the user's sets.
+  Adds device ip(s) to the user's sets, omitting missing IPs.
   """
   def add_device(device) do
     list_dev_sets(device.user_id)
-    |> Enum.each(fn set_spec -> add_elem(set_spec.name, device[set_spec.ip_type]) end)
+    |> Enum.filter(fn set_spec ->
+      # Only call add_elem/2 for IPs that are present
```

Comment:
We must have a test for this one

Reply:
This logic will be moved to the gateway in `0.8` and will be tested there.